plugins/modules/vmware_dvswitch.py

```diff
@@ -260,6 +314,13 @@
         "health_check_teaming_interval": 0,
         "health_check_vlan": false,
         "health_check_vlan_interval": 0,
+        "netFlow_collector_ip": "192.168.10.50",
+        "netFlow_collector_port": 50034,
+        "netFlow_observation_domain_id": 0,
+        "netFlow_active_flow_timeout": 60,
+        "netFlow_idle_flow_timeout": 15,
+        "netFlow_sampling_rate": 4096,
+        "netFlow_internal_flows_only": False,
```

Comment:
```suggestion
        "net_flow_collector_ip": "192.168.10.50",
        "net_flow_collector_port": 50034,
        "net_flow_observation_domain_id": 0,
        "net_flow_active_flow_timeout": 60,
        "net_flow_idle_flow_timeout": 15,
        "net_flow_sampling_rate": 4096,
        "net_flow_internal_flows_only": False,
```

Thanks for changing the result from `netFlow_*` to `net_flow_*`. But we should also update the documentation to reflect this ;-)